openfisca_uk/parameters/benefits/LHA/rate_caps/beds_1.yaml

```diff
@@ -0,0 +1,7 @@
+description: LHA maximum rate for accommodation in Maidstone, Kent
```

Comment:
Is LHA only for Maidstone, Kent? Is there an appropriate place to document the benefit at a high level, e.g. `benefits/LHA/README.md`? If nothing else to unabbreviate.

Reply:
We're only using the rates for Maidstone, Kent, following [TaxBEN's example](http://www.oecd.org/els/soc/TaxBEN-United-Kingdom-2018.pdf#page=12). I'm also thinking we should make separate documentation for the whole model?